types/aws-lambda/trigger/appsync-resolver.d.ts

```diff
@@ -73,6 +73,7 @@ export interface AppSyncAuthorizerEvent {
         accountId: string;
         requestId: string;
         queryString: string;
+        operationName: string;
```

Comment:
Is this always a string? Is it ever undefined?

Reply:
Just confirmed on my own infra, that operationName may be optional that may not exist. Updated the PR to reflect it's an optional field